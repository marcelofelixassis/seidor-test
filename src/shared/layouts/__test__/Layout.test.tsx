import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../layout';
test('renders the component', () => {
  const component = shallow(<Layout />);
  expect(component).toMatchSnapshot();
});