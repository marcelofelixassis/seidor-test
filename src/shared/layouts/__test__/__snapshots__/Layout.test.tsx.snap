// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div>
  <Header />
  <WithStyles(ForwardRef(Container))
    className="body-container"
    maxWidth="md"
  >
    <div />
  </WithStyles(ForwardRef(Container))>
</div>
`;
